### DISCUSSION TITLE
FN1 discussion

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/library#308] [General]
[Activity 2.1.4](https://library.tbil.org/2024/precalculus/FN1.html#FN1-exploring-relations-1)  - 2.1.6 go together and my students did well with them, especially in realizing that it might not be necessary to write things twice. I would recommend one additional exercise to go with this group though. 2.1.4 is one-to-one, 2.1.5 is not a function and also has repeated y values, and 2.1.6 is function but just isn't one-to-one. It might be good to throw in another non-function but no repeated y values. To help point out that it's because of the x-values having two different outputs.

I'd also recommend something to motivate why we care that an input has two outputs but not that the same output happened to come from two different inputs. I like to do this with a restaurant menu. Show them two potential menus and ask which one makes more sense or doesn't have any discrepancies if you were trying to order. Something like:

Menu A 

- Nuggets $2
- Burger $4
- Fries $2
- Nachos $5

Menu B

- Nuggets $2
- Burger $4
- Fries $3
- Nuggets $5

So the first one is a fine menu. We can accept that two different foods have the same price. But the second is a problem. How much do nuggets cost? Who knows?

**Comment by AbbyANoble** [comment]
The end of this section has some questions about finding domain and range of a function from the equation. I think one more set of fluency problems would be good. Questions that include some denominators and/or square roots. And I think this should be added as a CheckIt problem.

**Reply by AbbyANoble** [reply]
And add domain and range to the objective!

**Comment by tdegeorge** [comment]
I like the menu problem!
